text/bayes.go

```diff
@@ -160,6 +161,30 @@ type NaiveBayes struct {
 	Output io.Writer
 }
 
+// histogram allows conncurrency-friendly map access via its
+// exported Get and Set methods
+type histogram struct {
```

Comment:
I'd probably say no because it doesn't seem necessary to export and it'll be much easier in the future to export it if it becomes necessary than to worry about breaking people's code and removing that functionality